lib/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "gatsby-source-storyblok",
+  "contributors": [
+    {
+      "name": "Alexander Feiglstorfer <delooks@gmail.com>"
+    }
+  ],
```

Comment:
We can remove this block

Reply:
I think @alexjoverm and I added it to see if it passes when we trigger the GitHub actions. I can delete if the other SDKs don't contain👍

Reply:
Yep, Facundo is right